api/src/wfl/util.clj

```diff
@@ -82,6 +86,45 @@ vault.client.http/http-client                               ; Keep :clint eastwo
     (subs string 0 (- (count string) (count suffix)))
     string))
 
+(defn remove-extension
```

Comment:
This is very similar to `util/unsuffix`. Do we need to keep both?

Reply:
You need to know the suffix to `unsuffix` and thus they have a different intent.